# SetRobotState.srv
# Service to set the robot state

# Request
uint8 state

# State constants (must match RobotState.msg)
uint8 IDLING = 1
uint8 EXPLORING = 2
uint8 PLANNING = 3
uint8 NAVIGATING = 4
uint8 SEARCHING = 5
uint8 GRASPING = 6

---
# Response
bool success
string message
